indigo_app/templates/library.html

```diff
@@ -37,74 +37,52 @@
   <thead>
     <tr>
       <th class="title" data-sort="title">Title</th>
-      <th class="year" data-sort="year">Year / Number</th>
-      <th class="updated_at" data-sort="updated_at">Updated</th>
+      <th class="uri" data-sort="uri">Frbr uri</th>
+      <th class="langs" data-sort="langs">Languages</th>
+      <th class="psit" data-sort="psit">Points in time</th>
     </tr>
   </thead>
+
   <tbody>
     {{#each works}}
-      {{#unless docs}}
-        <tr>
-          <!-- work without any documents -->
-          <td>
+      <tr>
+
+        {{!-- Title --}}
+        <td>
+          {{#if work_docs}}
```

Comment:
This isn't strictly true, but let's leave it for now. I'm not yet sure how to indicate whether some expressions published/some are draft/all are draft/ etc.